test/test_torch.py

```diff
@@ -3469,6 +3490,30 @@ def run_test(x, y):
         y[1] = 1.
         run_test(x, y)
 
+    # Ensures that median throws nondeterministic alerts in the correct cases
+    @dtypes(torch.double)
+    def test_median_nondeterministic_alert(self, device, dtype):
+        def test_func(slf, device, call_type):
```

Comment:
This doesn't appear to be called anywhere?

Reply:
Whoops, I must have accidentally chopped off the calls when I rebased. I'll add them back in